config/jobs/cert-manager/cert-manager-periodics.yaml

```diff
@@ -435,7 +435,69 @@ periodics:
       - name: ndots
         value: "1"
 
-- name: ci-cert-manager-upgrade-v1-21
+- name: ci-cert-manager-e2e-v1-22
```

Comment:
This is not a new job. I swapped some job definitions in the file and git displays the change in a weird way.